tests/test_elf.py

```diff
@@ -789,8 +789,7 @@ def test_elf_linux_x8664_epoll_simple(self):
         ql.run()
 
         self.assertIn(b'echo\n', ql.os.stdout.read())
-        del ql
-    @unittest.skip("See comment in https://github.com/qilingframework/qiling/pull/1558")
+        del ql 
```

Comment:
Please bring back the blank lines, here and down below where they got [accidently?] removed.